app/src/main/res/layout/activity_event.xml

```diff
@@ -44,9 +44,17 @@
                 android:numStars="5"
                 android:rating="2.5"
                 app:layout_constraintBottom_toTopOf="@+id/event_divider"
-                app:layout_constraintEnd_toEndOf="@+id/img_event_logo"
                 app:layout_constraintStart_toStartOf="@+id/img_event_logo"
                 app:layout_constraintTop_toBottomOf="@+id/img_event_logo" />
+            <TextView
+                android:id="@+id/id_number_reviews"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toTopOf="@+id/ratingBar_event"
+                app:layout_constraintEnd_toEndOf="@+id/img_event_logo"
+                app:layout_constraintBottom_toBottomOf="@+id/ratingBar_event"
+                android:text="0"
```

Comment:
Try to pus the default value in a string so that we can easily modify both of them if needed (and we avoid warning)

Reply:
Ok